low-level/sensors/impl/generic/cpu_fault_sensor.py

```diff
@@ -27,8 +27,8 @@
 from framework.base.internal_msgQ import InternalMsgQ
 from framework.base.module_thread import SensorThread
 from framework.base.sspl_constants import DATA_PATH
-from framework.utils.conf_utils import (GLOBAL_CONF, SSPL_CONF, Conf,
-    SITE_ID_KEY, RACK_ID_KEY, NODE_ID_KEY, CLUSTER_ID_KEY)
+from framework.utils.conf_utils import (NODE_ID_KEY, Conf,
+                                SSPL_CONF, GLOBAL_CONF)
```

Comment:
The wrapped line should be with just 4 space indent.

Reply:
Done.